src/components/RunMap/index.jsx

```diff
@@ -34,6 +38,13 @@ const RunMap = ({
         const map = ref.getMap();
         if (map && IS_CHINESE) {
           map.addControl(new MapboxLanguage({ defaultLanguage: 'zh-Hans' }));
+          if (RoadLabelDisplay == false) { 
+            // todo delete layers
+            map.on('load', () => {
+              for(let layerId of layerList){
+                map.removeLayer(layerId);
```

Comment:
can we use layerList.forEach((layerId) => {map.removeLayer(layerId)})

Reply:
all done :grin: